### PR TITLE
[ALLUXIO-1779] Add logging to the free command in BaseFileSystem

### PR DESCRIPTION
**Pull request by qzhong0605**
Just a small fix.

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

File: core/client/src/main/java/alluxio/client/file/BaseFileSystem.java [hunk 1]
```diff
@@ -155,6 +155,7 @@ public void free(AlluxioURI path, FreeOptions options)
     FileSystemMasterClient masterClient = mContext.acquireMasterClient();
     try {
       masterClient.free(path, options);
+      LOG.info("Freed file " + path.getPath());
```
Comment (gpang) [inline core/client/src/main/java/alluxio/client/file/BaseFileSystem.java]: Can you also add `" with options " + options`?

**Comment by gpang** [comment]
Thanks for the PR @qzhong0605 ! I left a minor comment.

File: core/client/src/main/java/alluxio/client/file/BaseFileSystem.java (hunk 1 above)
Comment (qzhong0605) [inline core/client/src/main/java/alluxio/client/file/BaseFileSystem.java]: In order to keep the same with CreateDirectory or Open command code, are there need options??

**Comment by gpang** [comment]
LGTM

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**